### PR TITLE
refactor: phone module migrate

### PR DESCRIPTION
**Pull request by 00thirdeye00**
- phone module migration from class to functions within phone namespace

#682

### CODE REVIEW

File: tests/modules/phone/PhoneTest.cpp [hunk 1]
```diff
@@ -24,79 +25,79 @@ class PhoneTest : public Test
...
-    phoneNumber = Phone::number(format);
+    phoneNumber = number(format);
     ASSERT_NE(phoneNumber, format);
     ASSERT_TRUE(isStringNumericWithSpecialChars(phoneNumber));
 }
 
 TEST_F(PhoneTest, IMEIGeneration)
 {
-    auto imei = Phone::imei();
+    auto generatedImei = imei();
```
Comment (00thirdeye00) [inline tests/modules/phone/PhoneTest.cpp:59]: variable name changed in test case

File: tests/modules/phone/PhoneTest.cpp [hunk 2]
```diff
@@ -24,79 +25,79 @@ class PhoneTest : public Test
...
 
 TEST_F(PhoneTest, AreaCodeGeneration)
 {
-    const auto areaCode = Phone::areaCode();
-    ASSERT_TRUE(std::ranges::any_of(phone::areaCodes.begin(), phone::areaCodes.end(),
-                                    [areaCode](const std::string_view& code) { return code == areaCode; }));
+    const auto generatedAreaCode = areaCode();
+    ASSERT_TRUE(std::ranges::any_of(areaCodes.begin(), areaCodes.end(),
+                                    [generatedAreaCode](const std::string_view& code) { return code == generatedAreaCode; }));
 }
```
Comment (00thirdeye00) [inline tests/modules/phone/PhoneTest.cpp:103]: variable name changed in test case

File: tests/modules/phone/PhoneTest.cpp (hunk 1 above)
Comment (00thirdeye00) [reply tests/modules/phone/PhoneTest.cpp:59]: does this have to go in changelog?

File: tests/modules/phone/PhoneTest.cpp (hunk 2 above)
Comment (00thirdeye00) [reply tests/modules/phone/PhoneTest.cpp:103]: does this have to go in changelog?

File: tests/modules/phone/PhoneTest.cpp (hunk 1 above)
Comment (cieslarmichal) [reply tests/modules/phone/PhoneTest.cpp:59]: yup thats fine :)

File: tests/modules/phone/PhoneTest.cpp (hunk 2 above)
Comment (cieslarmichal) [reply tests/modules/phone/PhoneTest.cpp:103]: its fine

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
What about phone data?

**Comment by 00thirdeye00** [comment]
phone data is already in phone namespace

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**